curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9f2bff625af342023512c.md

```diff
@@ -28,7 +28,7 @@ assert.match(code, /clearButton\s*\.addEventListener\s*\(\s*('|"|`)click\1\s*/);
 You should pass `clearForm` as the second argument to `.addEventListener()`.
 
 ```js
-assert.match(code, /clearButton\s*\.addEventListener\s*\(\s*('|"|`)click\1\s*,\s*clearForm\s*/);
+assert.match(code, /clearButton\s*\.addEventListener\s*\(\s*('|"|`)click\1\s*,\s*clearForm[\s\n]*\)?;?\s*$/);
```

Comment:
```suggestion
assert.match(code, /clearButton\s*\.addEventListener\s*\(\s*('|"|`)click\1\s*,\s*clearForm\s*\)\s*;?\s*/);
```
I wouldn't use the `$`, that will stop test working if anything more is added by camper after the working solution for a step.